src/functions_framework/__init__.py

```diff
@@ -162,6 +163,11 @@ def view_func(path):
     return view_func
 
 
+def read_request(response):
+    flask.request.get_data()
```

Comment:
```suggestion
def read_request(response):
    """
    Force the framework to read the entire request before responding, to avoid
    connection errors when returning prematurely.
    """

    flask.request.get_data()
```